src/gui/preferences.c

```diff
@@ -553,7 +553,7 @@ void dt_gui_preferences_show()
 
   //open in the appropriate tab if currently in darkroom or lighttable view
   const gchar *current_view = darktable.view_manager->current_view->name(darktable.view_manager->current_view);
```

Comment:
does that return "lighttable" / "darkroom" untranslated?

Reply:
I'm afraid that doesn't work, because the following line finds the correct tab to open by looking for its name, which _is_ translated. We only use translated names in the tabs.

Reply:
I see ! Well, then your patch is ok.